### PR TITLE
Add Traditional Chinese, Fix Simplified Chinese

### PR DESCRIPTION
**Pull request by TEGTainFan**
Correction and additional translation

### CODE REVIEW

**Review by TimGoll — CHANGES_REQUESTED** [review]
Please add an entry to the changelog, thanks!

Thank you for your translation

**Comment by TimGoll** [comment]
Oh wow, another language? :D Even more awesome!
(your new language misses a new line at the end of the file though

**Comment by TimGoll** [comment]
Feel free to add credits to your contributions (take a look at previous contributions, something like `by xyz`)

**Review by TimGoll — APPROVED** [review]

**Comment by TEGTainFan** [comment]
Corrected translation

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 All notable changes to TTT2 will be documented here. Inspired by [keep a changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## Unreleased
-
+- Add Traditional Chinese Translation
```
Comment (saibotk) [inline CHANGELOG.md:6]: Please add this point under the Added section

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,8 @@
...
+- Revise and additions simplified Chinese
```
Comment (saibotk) [inline CHANGELOG.md:7]: Add this under the Changed section

**Review by saibotk — CHANGES_REQUESTED** [review]
Thank you for your hard work. Just some minor requests in the changelog.

**Review by saibotk — APPROVED** [review]

**Pull request merged**